socorro/unittest/conftest.py

```diff
@@ -144,3 +152,26 @@ def boto_helper():
     """
     with BotoHelper() as boto_helper:
         yield boto_helper
+
+
+@pytest.fixture
+def es_conn():
+    """Create an Elasticsearch ConnectionContext and clean up indices afterwards.
+
+    This uses defaults and configuration from the environment.
+
+    """
+    manager = ConfigurationManager(
+        ESConnectionContext.get_required_config(), values_source_list=[environment]
+    )
+    conn = ESConnectionContext(manager.get_config())
+
+    # Create two indexes--this week and last week
+    template = conn.config.elasticsearch_index
+    conn.create_index(utc_now().strftime(template))
+    conn.create_index((utc_now() - datetime.timedelta(weeks=1)).strftime(template))
+    conn.refresh()
```

Comment:
Indexing crash reports will create missing indexes, but when setting up a test, it's easier if the test harness does it. Another way to do this that wouldn't have required me to fix the connection_context tests would be to create an ESHelper like we have the BotoHelper which has convenience methods for doing things. I like that idea a lot, but I'm sort of rushing this work out so I didn't want to spend time on that.

It's definitely a good future direction. It'll clean up a chunk of test code and make it much easier to remove a whole bunch of mocking.